tests/baselines/reference/ModuleWithExportedAndNonExportedFunctions.errors.txt

```diff
@@ -1,4 +1,4 @@
-tests/cases/conformance/internalModules/exportDeclarations/ModuleWithExportedAndNonExportedFunctions.ts(28,13): error TS2339: Property 'fn2' does not exist on type 'typeof A'.
+tests/cases/conformance/internalModules/exportDeclarations/ModuleWithExportedAndNonExportedFunctions.ts(28,13): error TS2551: Property 'fn2' does not exist on type 'typeof A'. Did you mean 'fn'?

```

Comment:
This should not have changed. The lower-bound cutoff was specifically designed to leave deletions from 3-character identifiers alone since they are really hard to miss visually.

(The same is almost as true of 4-character identifiers, as on the next line, but I remember coming up with a couple of convincing counterexamples.)